docs/excel/custom-functions-debugging.md

```diff
@@ -0,0 +1,157 @@
+---
+ms.date: 02/26/2019
+description: Debug your custom functions in Excel.
+title: Custom functions debugging (preview)
+localization_priority: Normal
+---
+# Custom functions debugging (preview)
+
+Debugging for custom functions can be accomplished by multiple means, depending on what platform you're using. Methods differ between [Office Online for Windows](#windows-10-visual-studio-and-microsoft-edge), [Office Online for Mac](#mac-and-chrome-debugger), [Office Desktop for Windows](#for-windows-and-visual-studio), and [Office Desktop for Mac](#for-mac). You can also issue debugging commands through [the command line directly](#using-the-command-line).
+
+[!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
+
+> [!NOTE]
+> For simplicity, this article shows debugging in the context of using Visual Studio Code to edit, run tasks, and in some cases use the debug view. If you are using a different editor or command line tool, see the [command line instructions](#Use-the-command-line-tools-to-debug) at the end of this article.
+
+## Use the browser developer tools to debug Excel Online
```

Comment:
this title is a bit confusing to me. I am not debugging Excel Online, I am debugging custom functions using my browser. should not be "Debugging Custom Functions using Browser"?  or something along those lines?